migrations.json

```diff
@@ -0,0 +1,192 @@
+{
+  "migrations": [
+    {
+      "cli": "nx",
```

Comment:
This whole file was accidentally committed by me, should be deleted!

Reply:
Done